consensus/core/src/ancestor.rs

```diff
@@ -61,7 +61,7 @@ pub(crate) struct AncestorStateManager {
 
 impl AncestorStateManager {
     // Number of quorum round updates for which an ancestor is locked in the EXCLUDE state
-    // Chose 10 updates as that should be ~50 seconds of waiting with the current round prober
+    // Chose 10 updates as that should be ~50 seconds of waiting with the current round tracker
```

Comment:
Continuing to use 5s for the QuorumRoundManager update interval but we can reduce this down to 1s now that it is decoupled from RoundProber. Only place that it would really effect is ancestor selection as propagation delay is based on received rounds and would still be tied to the RoundProber probe interval for any updated data.